packages/flutter_tools/gradle/flutter.gradle

```diff
@@ -80,82 +80,13 @@ class FlutterPlugin implements Plugin<Project> {
     // to match.
     static final String FLUTTER_BUILD_PREFIX = "flutterBuild"
 
-    private Path baseEnginePath
+    private Map baseJar = [:]
     private File flutterRoot
     private File flutterExecutable
     private String localEngine
     private String localEngineSrcPath
     private Properties localProperties
     private File flutterJar
-    private File debugFlutterJar
-    private File profileFlutterJar
-    private File releaseFlutterJar
-    private File dynamicProfileFlutterJar
-    private File dynamicReleaseFlutterJar
-
-    private Properties readPropertiesIfExist(File propertiesFile) {
```

Comment:
nit: Not a huge deal, but it's much easier to review and land patches if refactoring and actual logic changes are separated into multiple PRs. It's hard to know for sure here that this is just moving the functions around and not an intended change in behavior, and if this patch needs to be reverted based on the new logic being buggy this refactor will end up being reverted too.

Reply:
Happy to take the fix out of the PR since (that is a small delta after refactoring), and fix the bug later. Wdyt?

Reply:
resolved offline, this PR is about refactoring.